strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/layout/nuget/NugetArtifactController.java

```diff
@@ -406,9 +406,9 @@ private String getFeedUri(HttpServletRequest request, String storageId, String r
     @ApiOperation(value = "Used to check storage availability")
     @ApiResponses(value = { @ApiResponse(code = HttpURLConnection.HTTP_OK, message = "Storage available."),
                             @ApiResponse(code = HttpURLConnection.HTTP_UNAUTHORIZED, message = "Storage requires authorization.") })
-    @GetMapping(path = { "{storageId}/{repositoryId}", "greet" })
-    @PreAuthorize("hasAuthority('ARTIFACTS_DEPLOY')")
-    public ResponseEntity<String> greet()
+    @GetMapping(path = { "{storageId}/{repositoryId}", "checkRepositoryHealth" })
+    @PreAuthorize("hasAuthority('ARTIFACTS_RESOLVE')")
+    public ResponseEntity<String> checkRepositoryHealth()
```

Comment:
@anki2189 this probably better to be named `checkRepositoryAccess`

Reply:
Changed to `checkRepositoryAccess`